libs/analytics/src/lib/components/map/map.component.scss

```
@@ -1,23 +1,40 @@
+@import '@angular/flex-layout/mq';
+
 :host {
   display: block;
 }
 
 header {
   display: flex;
   gap: 8px;
-
-  h5 {
-    display: flex;
-  }
+  justify-content: space-between;
 }
 
 world-map {
+  flex: 1 1 auto;
   height: 300px;
   border-radius: 4px;
 }
 
-section {
+.map-wrapper {
   width: 100%;
+  flex: 1 1 100%;
+  display: flex;
+  flex-direction: row-reverse;
+  flex-wrap: wrap;
+  gap: 24px;
+  margin-bottom: 24px;
+  position: relative;
+}
+
+@include layout-bp(lt-md) {
+  .map-wrapper {
+    flex-direction: column;
+  }
+}
+
+mat-list.horizontal {
+  flex: 0 0 clamp(20%,300px,30%);
```

This can be more readable.
```suggestion
mat-list.horizontal {
  width: 30%;
}

@include layout-bp(lt-md) {
  width: 100%;
}
```
With this I can instantly see what the UI should look like